### DISCUSSION TITLE
在最新的路由规范里使用渲染模板的问题

### DISCUSSION THREAD

**Discussion by axojhf** [DIYgod/RSSHub#17137] [Q&A] [answered]
看过文档，发现最新的路由规范里没有提及到渲染模板，相关信息在**v2 路由规范**里能找到
那么如果我准备修改以前的路由到最新的规范，渲染模板是保留还是手工合并到js代码里，还是有什么其他更好的处理方法

**Comment by TonyRL** [comment, marked as answer]
Keep using art-template.

**Reply by axojhf** [reply]
Thanks
